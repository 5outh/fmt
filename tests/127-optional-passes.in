<?php
//passes:OrderUseClauses,AddMissingCurlyBraces,SmartLnAfterCurlyOpen,NormalizeLnAndLtrimLines,MergeParenCloseWithCurlyOpen,MergeCurlyCloseAndDoWhile,MergeDoubleArrowAndArray,ExtraCommaInArray,ResizeSpaces,Reindent,ReindentColonBlocks,ReindentLoopColonBlocks,ReindentIfColonBlocks,ReindentObjOps,EliminateDuplicatedEmptyLines,AlignEquals,AlignDoubleArrow,LeftAlignComment,RTrim,SettersAndGettersPass,ConstructorPass,OrderMethod,NormalizeLnAndLtrimLines,Reindent


class A {
	private $a;
	public $b;
	protected $c;
}
