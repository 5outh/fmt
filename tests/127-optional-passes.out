<?php
//passes:OrderUseClauses,AddMissingCurlyBraces,SmartLnAfterCurlyOpen,NormalizeLnAndLtrimLines,MergeParenCloseWithCurlyOpen,MergeCurlyCloseAndDoWhile,MergeDoubleArrowAndArray,ExtraCommaInArray,ResizeSpaces,Reindent,ReindentColonBlocks,ReindentLoopColonBlocks,ReindentIfColonBlocks,ReindentObjOps,EliminateDuplicatedEmptyLines,AlignEquals,AlignDoubleArrow,LeftAlignComment,RTrim,SettersAndGettersPass,ConstructorPass,OrderMethod,NormalizeLnAndLtrimLines,Reindent

class A {
	private $a;
	public $b;
	protected $c;
	private function getA(){
		return $this->a;
	}
	public function getB(){
		return $this->b;
	}
	protected function getC(){
		return $this->c;
	}
	private function setA($a){
		$this->a = $a;
	}
	public function setB($b){
		$this->b = $b;
	}
	protected function setC($c){
		$this->c = $c;
	}
	function __construct($a, $b, $c){
		$this->setA($a);
		$this->setB($b);
		$this->setC($c);
	}
}
